database/factories/EventPageUpdateFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Database\Factories;
+
+use App\EventPageUpdate;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class EventPageUpdateFactory extends Factory
```

Comment:
Certainly no objection to having these factories but I was wondering if they are necessary? Does defining their default state as `[]` here do something more than just calling `Blah::create()` (like we do in the prod code

Reply:
That's a very good question. Up until now I blindly copied what was done before without thinking too much about it, but I guess it would work like you describe as well (i.e. factory-less).